bootstrap-conda

```diff
@@ -131,6 +169,7 @@ echo >&2 $0:$LINENO: generate conda environment files
     (
         echo >&4 "  - pip:"
         echo >&5 "  - pip:"
+        echo >&6 "  - pip:"
```

Comment:
Well, if that's by design (whose?), then that's a severe bug in it, and this is the fix.

Reply:
The bigger flaw is related to what we explained to you over in https://github.com/sagemath/sage/pull/36765#issuecomment-1826198353: 

Creating conda packages is a downstream activity. If we were to take the design idea that "all standard packages need to have a conda package" seriously, it would mean that we could never add a standard package in our capacity as the upstream project. 

But neither is this current practice (neither with conda nor with any other downstream package repository), nor has such a change of policy been established or even discussed anywhere. And certainly this change cannot come into existence by gaslighting PR authors.